sensor/common/clusterentities/store_test.go

```diff
@@ -209,19 +278,34 @@ func (s *ClusterEntitiesStoreTestSuite) TestMemoryAboutPast() {
 						s.True(len(result) == 0, "Should not find endpoint %q in tick %d.  Result: %v", endpoint, tickNo, result)
 					}
 				}
+
+				// Assert on container IDs
+				s.T().Logf("Historical container IDs (tick %d): %s", tickNo, prettyPrintHistoricalData(entityStore.historicalContainerIDs))
+				s.T().Logf("All container IDs (tick %d): %v", tickNo, maps.Keys(entityStore.containerIDMap))
+				for contID, shallExists := range tCase.containerIDsAfterTick[tickNo] {
+					result, found := entityStore.LookupByContainerID(contID)
+					if shallExists {
+						s.Truef(found, "expected to find contID %q in tick %d", contID, tickNo)
+						s.Equalf(contID, result.ContainerID, "Expected the result to have contID %q in tick %d. Result: %v", contID, tickNo, result)
+					} else {
+						s.Require().Falsef(found, "expected not to find contID %q in tick %d", contID, tickNo)
+						s.Empty(result.ContainerID)
+					}
```

Comment:
[nit] I think we don't have a case where the pod is not found right?

Reply:
Added in [839cbf4](https://github.com/stackrox/stackrox/pull/13155/commits/839cbf482e0096368abd2d3f88cdf457fe579b69)